deployment.rst

```diff
@@ -63,14 +63,16 @@ manually taking other steps (see `Common Post-Deployment Tasks`_).
 Using Platforms as a Service
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The specific deployment steps vary greatly from one service provider to another,
-so check out the dedicated article for the service of your choose:
+Using a Platform as a Service (PAAS) can be a great way to deploy your Symfony app
+quickly and easily. There are many PAAS - below are a few that work well with Symfony:
 
 .. toctree::
     :maxdepth: 1
-    :glob:
 
-    deployment/*
+    deployment/heroku
```

Comment:
Instead of linking to those dead pages here, could we instead link to the real tutorials on those third-party websites? If you do that, you need to add `:orphan:` at the beginning of the RST files to avoid the "this file is not linked in any TOC" error (like here: https://github.com/symfony/symfony-docs/blob/master/bundles/index.rst)